ci/h2spec.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+shopt -s extglob
+
+SEP="$(tput bold)⣷$(tput sgr0)"
+LOGFILE=$(mktemp -t l5d-h2spec.log)
+SPIN='⣾⣽⣻⢿⡿⣟⣯⣷'
+
+spin() {
+    local pid=$!
+    local i=0
+    while [ "$(ps a | awk '{print $1}' | grep $pid)" ]; do
+        i=$(( (i+1) %8 ))
+        printf "\r$(tput bold)${SPIN:$i:1}$(tput sgr0)"
+        sleep .1
+    done
+}
+
+echo "${SEP} running h2spec against linkerd..."
+# test if we have a h2spec binary - either in the working dir,
+# or on the PATH
+set +e
+WHICH_H2SPEC=$(which h2spec)
+if [[ $? -eq 0 ]]; then
+    # if which found an h2spec executable, just return that path.
+    H2SPEC_EXEC="${WHICH_H2SPEC}"
+    set -e
+else
+    set -e
+    # which didn't find a h2spec on the path. check if it exists in
+    # the working dir, and if not, dow nload it.
+    if [ -e "h2spec" ] ; then
+        # h2spec was found in the working dir, use that.
```

Comment:
i'd say forego this step, as i would not expect h2spec to be in the linkerd dir.